### ISSUE TITLE
About training loss

### ISSUE THREAD

**Issue by Vp-SoLo** [GaussianCube/GaussianCube#6]
Thank you for your excellent work!
I am planning to use GaussianCube on my own dataset. May I know how far should the loss function drop? How much does the loss for unconditional generation drop in the paper?

**Comment by GaussianCube** [comment]
Hi, 
Thank you for reaching out and for your interest in using GaussianCube on your dataset. Regarding the loss curve for unconditional generation on ShapeNet Car, please find it displayed below.  If you have any further questions or need assistance, feel free to ask. 😊
![shapenet_car_0_850k](https://github.com/GaussianCube/GaussianCube/assets/164283176/4a6858b5-0511-454a-a8d8-4cb6f5df38a1)
![shapenet_car_850k_1000k](https://github.com/GaussianCube/GaussianCube/assets/164283176/411a8174-032e-4027-bab8-7412b2cf4891)
